log-and-debug.md

```diff
@@ -113,3 +114,27 @@ public function boot()
 
 Tip given by [@mmartin_joo](https://twitter.com/mmartin_joo/status/1473262634405449730)
 
+### Discover all events fired in one request
+
+If you want to implement a new listener to especific event but you dont know what is your name, u can log all events fired during the request.
+
+You can use the `\Illuminate\Support\Facades\Event::listen()` method on `boot()` method of `app/Providers/EventServiceProvider.php` to catch all events fired.
+
+> Important: Do not use the Log::* methods to not make a infinite loop (or skip all Log::* events)
```

Comment:
```suggestion
**Important:** If you use the `Log` facade within this event listener then you will need to exclude events named `Illuminate\Log\Events\MessageLogged` to avoid an infinite loop. 
(Example: `if ($event == 'Illuminate\\Log\\Events\\MessageLogged') return;`)
```